backend/endpoints/rooms.py

```diff
@@ -262,3 +262,56 @@ async def close_conversation(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to close conversation",
     )
+
+
+@router.get(
+    "/org/{org_id}/rooms/{room_id}/members",
+    response_model=ResponseModel,
+    status_code=status.HTTP_200_OK,
+    responses={
+        424: {"detail": "Failure to retrieve room members"},
+    },
+)
+async def get_members(org_id: str, room_id: str):
+    """
+    Get room members.
+    Returns all members in a room if the room is found in the database
+    Raises HTTP_424_FAILED_DEPENDENCY if there is an error retrieving the room members
+    Args:
+        org_id (str): A unique identifier of an organisation
+        room_id (str): A unique identifier of the room
+    Returns:
+        HTTP_200_OK (Room members)
+
+        {
+            "status": "success",
+            "message": "Room Members",
+            "data": {
+                "61696f5ac4133ddaa309dcfe": {
+                "closed": false,
+                "role": "admin",
+                "starred": false
+                },
+                "6169704bc4133ddaa309dd07": {
+                "closed": false,
+                "role": "admin",
+                "starred": false
+                }
+            }
+        }
+
+    Raises:
+        HTTPException [424]: Failed to retrieve room members
+    """
+    members = await get_room_members(org_id, room_id)
+
+    if members and members.get("status_code") is None:
+        return JSONResponse(
+            content=ResponseModel.success(data=members, message="Room Members"),
+            status_code=status.HTTP_200_OK,
+        )
+        
+    raise HTTPException(
+        status_code=status.HTTP_424_FAILED_DEPENDENCY,
+        detail="Failed to retrieve room members",
+    )
```

Comment:
This exception is also raised when an invalid room is provided. 

We should consider this special case and return a "room doesn't exist" message instead.

Reply:
I switched it to get room instead so room doesn't exist is accounted for as well as accounting for the failed dependency exception.